### PR TITLE
categories template

### PR DESCRIPTION
**Pull request by AsmahanM**

### CODE REVIEW

File: database/model.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import db from "./connection";
 
 export function getCategories() {
-  const GET_CATEGORIES = `SELECT plant_kind FROM plant_type`;
+  const GET_CATEGORIES = `SELECT * FROM plant_type`;
```
Comment (PJSalter) [inline database/model.js:4]: Great work on adding the asterix to select all from the different plants 🪴 🌷 🌱

File: pages/index.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import Head from "next/head";
+import Link from "next/link";
 import Image from "next/image";
 import styles from "../styles/Home.module.css";
-import {getCategories} from '../database/model.js'
+import { getCategories } from "../database/model.js";
```
Comment (PJSalter) [inline pages/index.js:3]: easier to read with the spaces between get categories and double quotes is a great spot ✨ 🪴 🌱

File: pages/index.js [hunk 3]
```diff
@@ -17,6 +18,25 @@ export default function Home({ categories }) {
   return (
     <div>
       <h1>Little Garden</h1>
+      <section className="categories">
+        <ul>
+          {categories.length ? (
+            categories.map((category) => (
```
Comment (PJSalter) [inline pages/index.js:25]: really like the map function used which will take in each category types 🪴

File: pages/index.js [hunk 4]
```diff
@@ -17,6 +18,25 @@ export default function Home({ categories }) {
...
+              <Link href={`/categories/${category.id}`} key={category.id}>
+                <a>
+                  <li>
+                    <div>
+                      <h2>{category.plant_kind}</h2>
```
Comment (PJSalter) [inline pages/index.js:30]: really like the structure, good size for category listings ✨

**Review by mariaalouisaa — APPROVED** [review]

File: pages/plants/garden.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+export default function Garden() {
+  return (
+    <>
+      <h1>Choose you garden plants</h1>
+      <h2>
+        <Link href="/">
```
Comment (PJSalter) [inline pages/plants/garden.js:8]: great syntax definition for link component ✨

**Review by PJSalter — APPROVED** [review]

**Pull request merged**